web/src/main.js

```diff
@@ -13,6 +16,8 @@ import FAQ from './components/faq';
 import TimeSlider from './components/timeslider';
 import LanguageSelect from './components/languageselect';
 
+'use strict';
```

Comment:
By convention 'use strict' should be added at the beginning of a script/function